proc/proc.go

```diff
@@ -343,9 +345,15 @@ func (dbp *Process) setChanRecvBreakpoints() (int, error) {
 				}
 				return 0, err
 			}
+			if _, ok := setBPs[ret]; ok {
+				// Do not set duplicate breakpoint in case if multiple
+				// goroutines waiting on the same channel
+				continue
+			}
 			if _, err = dbp.SetTempBreakpoint(ret); err != nil {
 				return 0, err
```

Comment:
Instead off keeping track of what we've set, let's just check for `err.(BreakpointExistsError)` and ignore it.